libraries/util/appCommon/adminKpc.py

```diff
@@ -1244,4 +1244,157 @@ def searchCustomer(page, strEmail):
+        param: strTotalRewards: Text
+        returns strTotalRewards: Text
+        Author: abernal_20240319
+        """
+        intTotalRewards = uCommon.getElemText(page, pAdmin.cu.totalRewardValueLbl)
+        return intTotalRewards
+    
+    @uCommon.ufuncLog  
+    def getTotalCreditsValue(page, strTotalCredits):
+        """ 
+        Objective: To get the Total Credits value.
+        param strTotalCredit: Text
+        returns strTotalCredit: Text
+        Author: abernal_20240319
+        """
+        strTotalCredits = uCommon.getElemText(page, pAdmin.cu.totalCreditValueLbl)
+        return strTotalCredits
+    
+    @uCommon.ufuncLog  
+    def deductTotalRewardValue(page, strValue):
+        """ 
+        Objective: To deduct rewards from the Total Rewards.
+        param strValue: Text
+        returns: None
+        Author: abernal_20240319
+        """
+        strTotalReward = uCommon.getElemText(page, pAdmin.cu.totalRewardValueLbl)
+        uCommon.waitAndClickElem(page, pAdmin.cu.totalRewardEditBtn)
+        strValue = int(strValue) + 100
+        strNewValue = '-' + str(strValue)
+        uCommon.waitAndSetElem(page, pAdmin.cu.rewardEditLbl, strNewValue)
+        uCommon.waitAndClickElem(page, pAdmin.cu.editEnterBtn)
+        uCommon.waitElemToBeVisible(page, pAdmin.cu.confirmationModalLbl)
+        uCommon.waitAndClickElem(page, pAdmin.cu.confirmationYesBtn)
+        uCommon.waitElemToBeVisible(page, pAdmin.cu.confirmationDeductionModalLbl)
+        uCommon.waitAndClickElem(page, pAdmin.cu.closeConfirmationDeductBtn)
+        uCommon.waitAndClickElem(page, pAdmin.cu.cancelEditBtn)
+        strTotalReward = uCommon.getElemText(page, pAdmin.cu.totalRewardValueLbl)
+        strReward = '0'
+        if strTotalReward == strReward:
+            uCommon.log(1, f'User has no rewards.')
+        else:
+            uCommon.log(2, f'User has still rewards. Nothing was deducted.')
+            
+    @uCommon.ufuncLog  
+    def deductTotalCreditValue(page, strValue):
+        """ 
+        Objective: To deduct credits from the Total Credits.
+        param strValue: Text
+        returns: None
+        Author: abernal_20240322
+        """
+        strTotalCredit = uCommon.getElemText(page, pAdmin.cu.totalCreditValueLbl)
+        uCommon.waitAndClickElem(page, pAdmin.cu.totalCreditEditBtn)
+        strValue = int(strValue) + 100
+        strNewValue = '-' + str(strValue)
+        uCommon.waitAndSetElem(page, pAdmin.cu.creditEditLbl, strNewValue)
+        uCommon.waitAndClickElem(page, pAdmin.cu.editEnterBtn)
+        uCommon.waitElemToBeVisible(page, pAdmin.cu.confirmationModalLbl)
+        uCommon.waitAndClickElem(page, pAdmin.cu.confirmationYesBtn)
+        uCommon.waitElemToBeVisible(page, pAdmin.cu.confirmationDeductionModalLbl)
+        uCommon.waitAndClickElem(page, pAdmin.cu.closeConfirmationDeductBtn)
+        strTotalCredit = uCommon.getElemText(page, pAdmin.cu.totalCreditValueLbl)

```

Comment:
Line 1364 and line 1374, same lang ng ginagawa, better if gawan na lang ng function, ireturn na lang strTotaCredit.